data/json/professions.json

```diff
@@ -3381,6 +3381,50 @@
         }
     },
     {
+                "type": "profession",
+        "ident": "boxer",
+        "name": "Boxer",
+        "description": "You were training for the fight of your life before the Cataclysm struck. Now you fight just to keep yourself alive.",
+        "points": 3,
+        "traits": [
+            "PROF_BOXER"
+        ],
+        "skills": [
+            {
+                "level": 2,
+                "name": "melee"
+            },
+            {
+                "level": 2,
+                "name": "unarmed"
+            },
+            {
+                "level": 2,
+                "name": "dodge"
+            }
+        ],
+        "items": {
+            "both": [
+                "shorts",
+                "gloves_wraps",
+				"boxing_gloves",
+				"headgear",
```

Comment:
You used tabs here. The project uses 4 spaces for indentation.
Notepad++ has an option to automatically replace tabs with spaces.